frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:ListClusters",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:DescribeCluster",
+                "elasticmapreduce:ListInstances",
+                "elasticmapreduce:TerminateJobFlows",
+                "elasticmapreduce:SetTerminationProtection"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:elasticmapreduce:{AWS_REGION}:{AWS_ACCOUNT}:cluster/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "translate:StopTextTranslationJob",
+                "translate:ListTextTranslationJobs",
+                "translate:DescribeTextTranslationJob",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "StringEqualsIgnoreCase": {
+                    "iam:ResourceTag/system": "MLSpace"
+                }
+            },
+            "Action": [
+                "iam:DetachRolePolicy",
+                "iam:DeleteRole",
+                "iam:DeleteRolePolicy",
+            ],
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/MLSpace*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:ListRoles",
+                "iam:ListEntitiesForPolicy",
+                "iam:ListPolicyVersions",
+                "iam:ListAttachedRolePolicies",
+                "iam:GetRole",
+                "iam:GetPolicy"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:DeletePolicy",
+                "iam:DeletePolicyVersion",
+            ],
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:policy/MLSpace*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "StringEquals": {
+                    "iam:PassedToService": "sagemaker.amazonaws.com"
+                }
+            },
+            "Action": "iam:PassRole",
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/MLSpace*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "StringEquals": {
+                    "iam:PassedToService": "translate.amazonaws.com"
+                }
+            },
+            "Action": "iam:PassRole",
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/{MLSPACE_APP_ROLE_NAME}",
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+4. Click next and optionally add tags to this policy
+5. Click next again and enter a name for this policy. You can name the policy whatever you'd like (ex: systemPolicy) but ensure you remember it as you'll need it when creating the role.
```

Comment:
I would change the recommended name to have an `mlspace-` prefix unless we are intentionally recommending this to avoid collisions.

Reply:
Ehh I don't think it's something to worry about, in the other instructions (like for app role) we don't even give an example name. As long as the customer remembers what it is for the following steps, it doesn't really matter. Plus depending on the customer, this may be white-labeled and not be mlspace.